flow360/component/simulation/outputs/output_entities.py

```diff
@@ -0,0 +1,35 @@
+from typing import List, Literal, Tuple, final
+
+import pydantic as pd
+
+from flow360.component.simulation.framework.entity_base import EntityBase, EntityList
+from flow360.component.simulation.primitives import Surface
+
+
+@final
+class Slice(EntityBase):
+    _entity_type: Literal["SliceType"] = "SliceType"
+    slice_normal: Tuple[float, float, float] = pd.Field()
+    slice_origin: Tuple[float, float, float] = pd.Field()
+
+
+@final
+class Isosurface(EntityBase):
+    _entity_type: Literal["IsosurfaceType"] = "IsosurfaceType"
+    surface_field: str = pd.Field()
+    # TODO: Maybe we need some unit helper function to help user figure out what is the value to use here?
+    surface_field_magnitude: float = pd.Field(description="Expect scaled value.")
```

Comment:
Why the names are `surface field` and `surface field magnitude`? I think names of these parameters are incorrect. It is volume field of which we take isosurface, right? Shouldn't this be called just `field` and `field_magnitude` or `iso_value`? Isosurface of surface field is line, not surface.

I'd suggest to rename them.

Reply:
Done